ee/clickhouse/models/cohort.py

```diff
@@ -341,7 +341,9 @@ def recalculate_cohortpeople_with_new_query(cohort: Cohort) -> Optional[int]:
     count = sync_execute(
         f"""
         SELECT COUNT(1)
```

Comment:
Oooooh, dang it.

`SELECT COUNT(DISTINCT id) FROM person` works too?

Reply:
...kinda sus though, I'd assume `id`s we get would be unique, based on the query 🤔

Reply:
.. Ah, nevermind, reading the test.. I see now, I didn't consider the shape of the table correctly.